osf/migrations/0121_remove_support_page_waffle_flag.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-07-19 16:51
+from __future__ import unicode_literals
+from waffle.models import Flag
+from django.db import migrations
+
+
+def reverse_func(state, schema):
+    flag = Flag(name='ember_support_page', everyone=False)
+    flag.save()
+
+
+def remove_support_page_waffle_flags(state, schema):
+    Flag.objects.get(name='ember_support_page').delete()
```

Comment:
This will error on environments that don't have this flag created. I'm not actually sure we need this migration, since there was no migration that created the flag (and therefore no guarantee that it exists).

Instead, the app code just needs to work whether the flag is there or not.

Reply:
Alternatively, you could make this just not error, as in https://github.com/CenterForOpenScience/osf.io/pull/8557/files#r210406496 .

Reply:
@sloria, these were added in this migration: https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/migrations/0083_add_ember_waffle_flags.py

Reply:
Ah, I missed that migration. Thanks @caseyrollins. In that case, keeping this migration makes sense.

Reply:
Yep. It's worth nothing that not all waffle flags are created via a migration, but these initial ones were 👍

Reply:
Since flags can be deleted via the admin app, there's still no guarantee that the flag exists, so we should still use `.filter` instead of .get` in the future.

No biggie, though--merged as is.